config.sample.js

```diff
@@ -0,0 +1,6 @@
+export const useLocales = [
+  'locales/default', // default locales
+  ['locales/ppy.sb', {
```

Comment:
this has to be removed?

Reply:
(The locales/ppy.sb, I hate github for showing the other lines)

Reply:
it's a sample to show how to merge different translations, let's keep it as it is. I an planning to give an example for default locales in readme